<?xml version="1.0"?>

<launch>
	<group ns="instance1">
		<node name="server_node" pkg="ros_tutorials" type="server_node.py" output="screen"/>
		<node name="client_node" pkg="ros_tutorials" type="client_node.py" output="screen"/>
	</group>
</launch>